src/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/StdModule.php

```diff
@@ -374,6 +374,43 @@ protected function getConfig(string $name, $default = false): string
         return $configurationValue;
     }
 
+    /**
+     * Gets the configuration value for a given configuration key directly from
+     * the database. This is usually not necessary, unless you have a specific
+     * reason, please use `getConfig` instead.
+     *
+     * @param string $name    The key of the configuration.
+     * @param mixed  $default The default value to return if the configuration
+     *                        key is not defined.
+     *
+     * @see getConfig() Gets the configuration.
+     *
+     * @return mixed The configuration value for the specified key, or the
+     *               default value if the key is not defined. Once the PHP
+     *               requirements go up to >= 8.0, we can return `string|false`
+     *               instead.
+     */
+    protected function getConfigFromDb(string $name, $default = false): mixed
```

Comment:
Since `mixed` was only introduced with PHP 8, but the standard module is used by many modules that unfortunately only run under PHP 7.4, we cannot use `mixed` as a type hint. See: https://www.php.net/manual/en/language.types.declarations.php

Reply:
Oh yikes, sorry!

What's a good alternative then? Do we have to return a string, or should we leave the type hint out entirely?

Reply:
String is a good choice, so does `StdModule::getConfig(): string`